docs/src/Examples/Guides/ControllingProgrammatically.jsx

```diff
@@ -31,7 +31,7 @@ class ControllingProgrammaticallyExample extends PureComponent {
         <div className="picker">
           <DatePicker
             clearable
-            pickerRef={(node) => { this.picker = node; }}
+            ref={(node) => { this.picker = node; }}
```

Comment:
Is that a breaking change? If so, we should mention that in release notes

Reply:
@cherniavskii thanks for good point. Surely will mention